build/InternalChecksCI-azureBuild-pipeline.yml

```
@@ -0,0 +1,175 @@
+name: fhir-ci-$(Date:yyyyMMdd)$(Rev:-r)
+
+parameters:
+- name: BuildConfiguration
+  default: release
+
+variables:
+- name: vmImage_1
+  value: Hosted Windows 2019 with VS2019
+- name: vmImage_2
+  value: windows-2019
+
+trigger:
+
+  branches:
+    include:
+    - main
+  paths:
+    include:
+    - /
+    exclude:
+    - /build
+    - /docs
+  batch: True
+
+resources:
+  repositories:
+  - repository: self
+    type: git
+    ref: main
+
+jobs:
+
+- job: Phase_1
+  displayName: Internal Checks CI - build and restore
+  cancelTimeoutInMinutes: 1
+  pool:
+    name: $(vmImage_1)
+  steps:
+  - checkout: self
+    persistCredentials: True
+  - task: NuGetToolInstaller@0
+    displayName: Use NuGet 4.3.0
+    inputs:
+      versionSpec: 4.3.0
+
+  - task: NuGetCommand@2
+    displayName: NuGet restore
+    enabled: False
+    inputs:
+      selectOrConfig: config
+      nugetConfigPath: nuget.config
+
+  - task: UseDotNet@2
+    displayName: Use .NET Core sdk
+    inputs:
+      version: 5.0.400
```

Use global.json instead

```suggestion
  - task: UseDotNet@2
    displayName: Use .NET Core sdk
    inputs:
      useGlobalJson: true
```

---

Done